src/plugins/data_views/docs/openapi/paths/api@data_views@default.yaml

```diff
@@ -60,6 +65,12 @@ post:
             properties:
               acknowledged:
                 type: boolean
+    '400':
```

Comment:
@mattkime And again here: https://www.elastic.co/guide/en/kibana/current/data-views-api-default-set.html